### ISSUE TITLE
PVR addon version number not being updated

### ISSUE THREAD

**Issue by emveepee** [xbmc/xbmc#18360]
## Bug report
After a success version update Kodi still reports that the addon still needs to be update
### Describe the bug
The internal version check still thinks a version update is required
```
2020-09-01 15:34:13.409 T:12856   DEBUG <general>: ADDONS: update check: addonID = pvr.nextpvr / Origin = repository.xbmc.org
2020-09-01 15:34:13.409 T:12856   DEBUG <general>: ADDONS: -- found -->: addonID = pvr.nextpvr / Origin = repository.xbmc.org / Version = 6.2.1
```

Here is a clear and concise description of what the problem is:

When starting Kodi a new version  of pvr.nextpvr is identified as being available.  I perform the update the addon restarts and the new files are installed.   Kodi still indicates that there is an available update for the newly installed version.  Users will likely redo the process thinking they did something work.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

The addon should disappear from the Available update list and show on Recently updated.

## Actual Behavior

No change on  Available updates or Recently updated screens

### To Reproduce

1.  Developer tags a build to the repo
2. User installs new version
3. User sees the screens below.
4.  User restarts Kodi and has the correct version

### Debuglog

The debuglog can be found here:

https://paste.kodi.tv/atuvejiday.kodi

### Screenshots 
Here are some links or screenshots to help explain the problem:

Available update after the update

![image](https://user-images.githubusercontent.com/2148031/91902718-25081380-ec70-11ea-9e75-c25d0fa7c852.png)

it also does not show in Recently updated

![image](https://user-images.githubusercontent.com/2148031/91902863-55e84880-ec70-11ea-92b7-e416fee640ae.png)

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

When entering the Addon screen before applying the first update.  The wrong version is shown at the top of the screen  It shows the found version not the installed version

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [ x] Windows
 - [ ] Windows UWP

 - Operating system version/name:
Win10 Home

 - Kodi version:

Kodi (19.0-ALPHA1 (18.9.701) Git:20200830-a3d5f73375)

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by ksooo** [comment]
@AlwinEsch I have the same problem with pvr.hts. Update installs fine, but "Updates available" lists the just updated addon as still not up to date. I have to restart Kodi to fix the problem.

**Comment by AlwinEsch** [comment]
Will see soon, PVR takes a different route, but possibly a similar cause as on VFS. Think if addon is still in use, its active class and addon data will be reused and will appear as the previous version.

In the event of an update, it must be ensured that the current add-on has ended.

**Comment by emveepee** [comment]
I am not sure if it is related to this issue but when I upgraded to alpha2 there were many other idiosyncrasies in updating.    The addon might says Origin: Manual when it from the repo or a troubling scenario when it gives only the Install option when it is installed and actually working.

Unfortunately the addon got disabled  on an API change rather then telling me about a new version so it not clear if this issue still exists.  I do have a new version coming soon where I will be able to update this specific issue.

**Comment by phunkyfish** [comment]
@howie-f found an issue with add-ons reporting as Manual when they are not. He's working on it currently. But this issue did not exists a month ago so I don't believe it's related.

**Comment by emveepee** [comment]
I did get a prompt that there is an available update but instead of getting the update button I have an install button

![image](https://user-images.githubusercontent.com/2148031/95351380-2b049c00-088f-11eb-8667-d6316842a6c6.png)


It is installed and working   I did not install in case there is something someone wants tested


![image](https://user-images.githubusercontent.com/2148031/95351585-5c7d6780-088f-11eb-9994-d935ae0c5c2f.png)

**Comment by howie-f** [comment]
@emveepee it is correct that you see the `Install` button for now. this is due to the recent addon rework and a gui change is pending that will turn this to an `Update` button

**Comment by emveepee** [comment]
Thanks I will keep this portable mode directory around  until I see a nightly with the fix.  Hopefully it closes the issue too.

**Comment by howie-f** [comment]
you can see some previews at #18527 and maybe you'd like to test it.
actually the `Install` button also processes updates even if the caption is not changed yet

**Comment by emveepee** [comment]
I saw that  I could update when i was playing around with alpha2 but it gave some pretty cryptic dialog.  The issue for me is that Install blocks Configure and not everyone wants to be forced to update.  

If it isn't too hard to mimic a clean before state" in my dev world I will try a test with the PR and post on the other thread.

**Comment by ksooo** [comment]
Fixed by #18627
